parsl/dataflow/dflow.py

```diff
@@ -1015,7 +1030,7 @@ def checkpoint(self, tasks=None):
                         # mode behave like a incremental log.
                         pickle.dump(t, f)
                         count += 1
-                        self.tasks[task_id]['checkpoint'] = True
+                        # self.tasks[task_id]['checkpoint'] = True
```

Comment:
this `checkpoint` field is referenced in the construction of the task record and also passed to monitoring - probably it should disappear from there?

Reply:
I will remove those.

Reply:
there are a few places where code is commented out when it should be removed